apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts

```
@@ -66,6 +66,8 @@ export class GiddhDatepickerComponent implements ControlValueAccessor, OnInit, O
     /** Placeholders for the callbacks which are later provided by the Control Value Accessor */
     private onTouchedCallback: () => void = noop;
     private onChangeCallback: (_: any) => void = noop;
+    /** This is used to show change date */
+    public inputChange: any = '';
```

Biome (1.9.4)

[error] Duplicate class member name "inputChange"



(lint/suspicious/noDuplicateClassMembers)